library/sl

```diff
@@ -8,7 +8,7 @@ Tags: 7, latest
 Architectures: amd64
 GitFetch: refs/heads/7
 Directory: sl7
-GitCommit: 1bc6a746a62e6ef13afc53aa585be9368fa3538b
+GitCommit: 9bf1a0c449251e6edf1f61d8f7a80a06e1d7fe13
```

Comment:
So this needs to be updated to `b1a2cb1c4a0efc1965f3344f8bdf1c949c7b79c2` now (https://github.com/scientificlinux/sl-docker/commits/7). :+1: